ZoomImageView/ZoomImageView.swift

```diff
@@ -31,7 +31,7 @@ open class ZoomImageView : UIScrollView, UIScrollViewDelegate {
 
   // MARK: - Properties
 
-  private let imageView = UIImageView()
+  public let imageView = UIImageView()
```

Comment:
@nuudles 
Does it need to change to public?

Reply:
No, I used that for my own purposes to expose the `imageView` to animate for a transition. Feel free to leave it as `private`.

Reply:
Okay.
then, could you change back this to private?